Tests/Fixtures/Generated/Strings/objc-m/localizable.m

```diff
@@ -14,11 +14,12 @@ @implementation BundleToken
 static NSString* tr(NSString *tableName, NSString *key, ...) {
 
     NSBundle *bundle = [NSBundle bundleForClass:BundleToken.class];
-    NSString *format = [bundle localizedStringForKey:key value:@"" table:tableName];
+    NSString *format = [bundle localizedStringForKey:key value:nil table:tableName];
+    NSLocale *locale = [NSLocale currentLocale];
 
     va_list args;
     va_start(args, key);
-    NSString *result = [[NSString alloc] initWithFormat:format arguments:args];
+    NSString *result = [[NSString alloc] initWithFormat:format locale:locale arguments:args];
```

Comment:
Good catch!

Had a doubt for a minute so had to [dig back that old discussion in old issue](https://github.com/SwiftGen/SwiftGen/issues/91) to ensure the difference between `locale: [NSLocale currentLocale]` vs `locale: nil` vs no locale parameter. And indeed the right solution is to explicitly pass current :+1:

Reply:
Yep, had to look up the exact same convo on Slack 😆